lib/ex_webrtc/ice/fly_ip_filter.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ExWebRTC.ICE.FlyIpFilter do
+  @moduledoc """
+  ICE IP filter for Fly.io deployments.
+
+  This module defines a single function, which filters out IP addresses,
+  which ICE Agent will use as its host candidates.
+  """
+
+  @spec ip_filter(:inet.ip_address()) :: boolean()
```

Comment:
[nitpick] We could add a behaviour in ExICE. Then again, it's only one function -- up to you

Reply:
Let's stay with the current version for know. In debugging and testing it's verry convenient to pass anonymous function instead of creating a module with a single function.

I thought about returning an anonymous function from `ip_filter` and use `ICEAgent.ip_filter` type in typespec but sounds like an overkill :thinking: